src/modules/auth/auth.controller.ts

```diff
@@ -11,6 +11,7 @@ export class AuthController {
     return this.authService.signUp(data);
   }
 
+  // what is the need for the first decorator?
```

Comment:
when i do a post request, the main purpose is to create something, so when the user sign in, there's no data created except the tokens, Post decorator in this case will send me a 201 response, which is created status, but in my case I think it's not.

let me know if I'm right.